server/mailet/integration-testing/src/test/java/org/apache/james/transport/mailets/GroupMappingTest.java

```diff
@@ -374,4 +395,48 @@ public void messageShouldRedirectToGroupWhenDomainMapping() throws Exception {
             calmlyAwait.atMost(Duration.ONE_MINUTE).until(() -> imapMessageReader.userReceivedMessage(USER_DOMAIN2, PASSWORD));
         }
     }
+
+    @Test
+    public void externalGroupMemberAreNotSupported() throws Exception {
```

Comment:
damned, we should. What's wrong with external members?

Reply:
They are filtered out in RRT processor. @quynhn , could you remove external address filtering (and correct tests that breaks?)

Reply:
Sorry but last commit please.